docs/guides/primitives.mdx

```diff
@@ -51,8 +51,8 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
 
 ## Benefits of Qiskit primitives
 
-For Qiskit users, primitives let you write quantum code for a specific quantum computer without having to explicitly
-manage every detail. In addition, because of the additional layer of abstraction, you might be able to more easily
+For Qiskit users, primitives let you write quantum code for a specific system without having to explicitly
```

Comment:
Just double-checking - Is this referring to something other than a QPU?

Reply:
Nope - I think it was lost in the shuffle